### ISSUE TITLE
Checkbox label text is not read out by NVDA with Chrome when hovering over label

### ISSUE THREAD

**Issue by lfdebrux** [alphagov/govuk-frontend#2278]
**Upstream bug:** https://github.com/nvaccess/nvda/issues/13879
**Date:** 2022-07-08
**Reported by:** Vanita Barrett-Smith @vanitabarrett

---
From conversation on x-gov Slack with @MalcolmVonMoJ 

## Description of the issue

Checkboxes only read out the label when you hover over the checkbox, not when you hover over the label (NVDA version 2018.1)

Checkboxes not announced at all when hovering over label or input (NVDA version 2019.1 and above)

## Steps to reproduce the issue

1. Open Assistiv labs with NVDA 2018.1 and Chrome latest
2. Go to https://govuk-frontend-review.herokuapp.com/components/checkboxes/preview
2. Enable NVDA mouse tracking with keyboard shortcut `NVDA + m`
3. Hover mouse over checkbox label(s)

## Actual vs expected behaviour

NVDA reads out the label text when hovering over the label.

## Environment

- Browser: Chrome
- Assistive tech: NVDA 2015

Can reproduce with NVDA versions 2018.1 with Chrome 91 (latest).

**Comment by lfdebrux** [comment]
It's possible that this is a bug entirely with NVDA and Chrome; @36degrees found that it was possible to reproduce the issue with native checkboxes.

**Comment by lfdebrux** [comment]
Note: with the latest version of NVDA (2021.1) hover over checkboxes doesn't read the label for either input or label.

**Comment by vanitabarrett** [comment]
This has been reported to NVDA now https://github.com/nvaccess/nvda/issues/13879 . There seem to be a lot of related issues about various elements not being announced in Chrome when mouse tracking is enabled, so I've also linked to one of those cards from the bug report.

**Comment by querkmachine** [comment]
Had another poke at this in NVDA 2025.3 and Chrome 140 and yep, it's still an issue, but now it doesn't work for virtually _any_ element. OS and browser UI works fine, but virtually anything within a webpage (GOV.UK or not) is announced.

This seems to be consistent with the NVDA issue reported here, which our report has been folded into: https://github.com/nvaccess/nvda/issues/11845
